config/routes/framework.yaml

```
@@ -2,3 +2,31 @@ when@dev:
     _errors:
         resource: '@FrameworkBundle/Resources/config/routing/errors.xml'
         prefix: /_error
+about:
+    path: /about
+    controller: App\Controller\AboutController::index
+    methods: GET
+
+createBlog:
+    path: /createBlog
+    controller: App\Controller\AboutController::createBlog
+    methods: GET
+
+
+Registration:
+    path: /Registration
```

Якщо це не API ендпоінт, шляхи краще вказаувати з маленької букви
```suggestion
    path: /registration
```